modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosControllerEpics.scala

```diff
@@ -230,47 +229,41 @@ object GmosControllerEpics extends GmosEncoders {
   def setWavelength(state: GmosCCEpicsState, w: Length): Option[IO[Unit]] =
     applyParam(state.disperserWavel, encode(w), CC.setDisperserLambda)
 
-  def setDisperserParams[T <: GmosController.SiteDependentTypes: Encoders](state: GmosCCEpicsState, cfg: GmosController.Config[T], d: GmosController.Config[T]#GmosDisperser): List[Option[IO[Unit]]] = {
+  def setDisperserParams[T <: GmosController.SiteDependentTypes: Encoders](state: GmosCCEpicsState,
+                                                                           cfg: GmosController.Config[T],
+                                                                           d: GmosController.Config[T]#GmosDisperser)
+  : List[Option[IO[Unit]]] = {
     val set: List[Option[IO[Unit]]] = d match {
```

Comment:
The name `set` maybe a bit confusing, you imagine it is a function rather than a value